File: SPA/src/app/_core/_models/shortenedUrl.ts
export interface ShortenedUrl {
  id: string;
  originalUrl: string;
  shortCode: string;
  shortUrl: string;
  hits: number;
  createdAt: string;
}
export interface GetUrlShortened {
  originalUrl: string;
}

export interface AllShortenedUrl {
  count: number | null;
  countToday: number | null;
  countHits: number | null;
  listShortenedUrl: ShortenedUrl[] | null;
}

File: API/Models/ShortenedUrl.cs
namespace API.Models
{
    public class ShortenedUrl
    {
        public Guid Id { get; set; }
        public string OriginalUrl { get; set; } = default!;
        public string ShortCode { get; set; } = default!;
        public string ShortUrl { get; set; } = default!;
        public int Hits { get; set; }
        public DateTime CreatedAt { get; set; }
    }
}